src/storage/src/hummock/block_cache.rs

```diff
@@ -76,7 +75,11 @@ impl BlockCache {
         if capacity == 0 {
             panic!("block cache capacity == 0");
         }
-        let cache = LruCache::new(CACHE_SHARD_BITS, capacity, DEFAULT_OBJECT_POOL_SIZE);
+        let mut shard_bits = CACHE_SHARD_BITS;
```

Comment:
May want to rename `CACHE_SHARD_BITS` to `MAX_CACHE_SHARD_BITS`

Reply:
ok